Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestDetailsPage.tsx

```diff
@@ -234,10 +235,15 @@ const ContestDetailsPage = () => {
 
             return allowedSubmissionTypes.map((x) => (
```

Comment:
Is it possible to use List component instead of allowedSubmissionTypes here?

Reply:
I tried, but there is always something not right with the displaying of the strategies there, so I left it unchanged.